GroupManager.py

```diff
@@ -73,22 +73,23 @@ def __init__(self, bot: Bot):
     async def listGroupsCommand(self, context, groupName=None):
         messageToSend = ''
         if (groupName is None) or (groupName is RESERVED_WORDS) or (groupName not in groupData): # Conditions to list all groups (no name given, reserved words, or isn't in groupData)
-            messageToSend += 'There are currently ' + str(len(groupData)) + ' groups on PWPG!\n' 
+            messageToSend += 'There are currently ' + str(len(groupData)) + ' groups on PWPG!\n'
             for n in groupData:
                 messageToSend += n + ': '
                 if groupData[n]['description'] != 'No Description': messageToSend += groupData[n]['description'] # Add the description if the group has one
                 messageToSend += '\n\tMembers: ' + str(len(groupData[n]['member'])) + '\n'
         elif groupName in groupData:
             messageToSend += groupName + ' has ' + str(len(groupData[groupName]['member'])) + ' members.\n' # <groupName> has <number> members \n
             if groupData[groupName]['description'] != 'No Description': messageToSend += groupData[groupName]['description'] + '\n' # Add the description if the group has one
+            messageToSend += '---' + '\n'
             for m in groupData[groupName]['member']: # Add each member
-                messageToSend += str(m) + '\n'
+                messageToSend += str(groupData[groupName]['member'][m]['displayName']) + '\n'
```

Comment:
Could use [Member converter](https://discordpy.readthedocs.io/en/rewrite/ext/commands/api.html#discord.ext.commands.MemberConverter) to skip needing to write the displayName into the file. This way it would also stay up to date if a user changes their guild nickname.

Reply:
Makes much better sense, yeah